gplog/gplog.go

```diff
@@ -59,6 +60,16 @@ const (
 	LOGDEBUG
 )
 
+// --- constants for colorized output support
+const escape = "\x1b"
+
+const (
```

Comment:
Go's enum type is a little odd compared to other languages. The "idiomatic" method to go about it is to have a `type Color int` before the const `iota` block, such that any function that takes in these `ints` can be less confusing by taking in the custom `Color` type instead of the primitive `int` type.

An example here: https://go.dev/doc/effective_go#initialization